tests/acceptance/features/apiOcm/share.feature

```diff
@@ -695,4 +695,23 @@ Feature: an user shares resources using ScienceMesh application
           }
         }
       }
-      """
\ No newline at end of file
+      """
+
+  @issue-10285 @issue-10536
+  Scenario: local user upload file to a federated shared folder via TUS
+    Given using spaces DAV path
```

Comment:
does this matter about spaces dav path in this scenario?

Reply:
Yes it matters, as we need to hit endpoint with spaceId.
:https://github.com/owncloud/ocis/blob/9b8b92c72fdbbb81951defdffa896d22b05e33cb/tests/acceptance/bootstrap/TUSContext.php#L310

Reply:
we must specify space dav path for federation shares (doesn't work with other dav paths, see https://github.com/owncloud/ocis/issues/10536). by default new dav path is used in the tests